dotCMS/src/main/java/com/dotcms/graphql/AbstractGraphQLHttpServlet.java

```diff
@@ -1,446 +1,446 @@
-
-        respWriter.write(']');
-    }
-
-    private <R> List<R> runListeners(Function<? super GraphQLServletListener, R> action) {
-        return configuration.getListeners().stream()
-                .map(listener -> {
-                    try {
-                        return action.apply(listener);
-                    } catch (Throwable t) {
-                        log.error("Error running listener: {}", listener, t);
-                        return null;
-                    }
-                })
-                .filter(Objects::nonNull)
-                .collect(Collectors.toList());
-    }
-
-    private <T> void runCallbacks(List<T> callbacks, Consumer<T> action) {
-        callbacks.forEach(callback -> {
-            try {
-                action.accept(callback);
-            } catch (Throwable t) {
-                log.error("Error running callback: {}", callback, t);
-            }
-        });
-    }
-
-    private boolean isBatchedQuery(InputStream inputStream) throws IOException {
-        if (inputStream == null) {
-            return false;
-        }
-
-        ByteArrayOutputStream result = new ByteArrayOutputStream();
-        byte[] buffer = new byte[128];
-        int length;
-
-        inputStream.mark(0);
-        while ((length = inputStream.read(buffer)) != -1) {
-            result.write(buffer, 0, length);
-            String chunk = result.toString();
-            Boolean isArrayStart = isArrayStart(chunk);
-            if (isArrayStart != null) {
-                inputStream.reset();
-                return isArrayStart;
-            }
-        }
-
-        inputStream.reset();
-        return false;
-    }
-
-    private boolean isBatchedQuery(String query) {
-        if (query == null) {
-            return false;
-        }
-
-        Boolean isArrayStart = isArrayStart(query);
-        return isArrayStart != null && isArrayStart;
-    }
-
-    // return true if the first non whitespace character is the beginning of an array
-    private Boolean isArrayStart(String s) {
-        for (int i = 0; i < s.length(); i++) {
-            char ch = s.charAt(i);
-            if (!Character.isWhitespace(ch)) {
-                return ch == '[';
-            }
-        }
-
-        return null;
-    }
-
-    protected interface HttpRequestHandler extends BiConsumer<HttpServletRequest, HttpServletResponse> {
-        @Override
-        default void accept(HttpServletRequest request, HttpServletResponse response) {
-            try {
-                handle(request, response);
-            } catch (Exception e) {
-                throw new RuntimeException(e);
-            }
-        }
-
-        void handle(HttpServletRequest request, HttpServletResponse response) throws Exception;
-    }
-}
+//package com.dotcms.graphql;
+//
+//import com.google.common.io.ByteStreams;
+//import com.google.common.io.CharStreams;
+//
+//import org.slf4j.Logger;
+//import org.slf4j.LoggerFactory;
+//
+//import java.io.BufferedInputStream;
```

Comment:
why commented, sounds like it could be deleted at all

Reply:
Done